tools/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# ##################################################################################################
+# Reference File Checker
+# ##################################################################################################
+set(FLAY_REFERENCE_CHECKER_SOURCES reference_checker.cpp
+                                   ${P4TOOLS_SOURCE_DIR}/common/lib/logging.cpp
+)
+
+add_executable(flay_reference_checker ${FLAY_REFERENCE_CHECKER_SOURCES})
+target_link_libraries(flay_reference_checker ${FLAY_LIBS} libflay)
```

Comment:
Shall we put `libflay` before `${FLAY_LIBS}` since `libflay` depends on `${FLAY_LIBS}`? Not sure why this passed test actions, but I have to reorder them to compile the project.
(Sorry for late reply, an ffc cable in my laptop burned out and just got fixed.)

Reply:
Interesting, are you building this on a Mac? I can reorder it but I am surprised it makes a difference.

Reply:
I am building on a Ubuntu VM running on Windows

Reply:
I guess this is a similar issue as this [post](https://stackoverflow.com/a/16574344/14309461)